scripts/translate_ascii.py

```diff
@@ -51,7 +51,7 @@ def main() -> None:
         p.close()
         p.join()
 
-        write_all(inputs=tiffs, target=arguments.target)
+        write_all(inputs=tiffs, target=arguments.target, generated_name=False)
```

Comment:
Why don't we want to use the generated name here?

Reply:
Because the files that are being copied need to have their original name as they are not temporary files.